### DISCUSSION TITLE
Nginx更新失败

### DISCUSSION THREAD

**Discussion by motao123** [1Panel-dev/1Panel#5005] [Q&A]
### 1Panel 版本

专业版：v1.10.7-lts

### 问题描述

![微信截图_20240513181720.png](https://img.imotao.com/i/2024/05/13/6641e8c88891d.png)

### 重现步骤

1Panel-openresty-pUKb  | 2024/05/13 18:08:35 [warn] 14#14: no resolver defined to resolve zerossl.ocsp.sectigo.com while requesting certificate status, responder: zerossl.ocsp.sectigo.com, certificate: "/www/sites/xk7.cn/ssl/fullchain.pem"
1Panel-openresty-pUKb  | 2024/05/13 18:08:37 [error] 14#14: *59752 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 38.148.254.2, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:08:54 [error] 14#14: *59771 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:09:54 [error] 14#14: *59886 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:10:44 [error] 14#14: *59989 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 38.148.254.2, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:10:54 [error] 14#14: *60000 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:11:54 [error] 14#14: *60115 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:12:52 [error] 14#14: *60230 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 38.148.254.2, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:12:54 [error] 14#14: *60233 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:13:54 [error] 14#14: *60350 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:14:54 [error] 14#14: *60468 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:14:59 [error] 14#14: *60475 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 38.148.254.2, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:15:54 [error] 14#14: *60584 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:16:54 [error] 14#14: *60697 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 45.80.189.85, server: 0.0.0.0:443
1Panel-openresty-pUKb  | 2024/05/13 18:17:06 [error] 14#14: *60732 cannot load certificate "data:": PEM_read_bio_X509_AUX() failed (SSL: error:0909006C:PEM routines:get_name:no start line:Expecting: TRUSTED CERTIFICATE) while SSL handshaking, client: 38.148.254.2, server: 0.0.0.

**Comment by zhengkunwang223** [comment]
系统日志看一下升级日志
